### PR TITLE
Include configuration docs in readme

### PR DESCRIPTION
**Pull request by JohnMcLear**
Just some basic PR to help noobs like me :P

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,3 +12,7 @@ WinTile can be found on the GNOME Extension site:
 https://extensions.gnome.org/extension/1723/wintile-windows-10-window-tiling-for-gnome/
 
 <img src='demo.gif'>
+
+# Configuration
+1. Visit https://extensions.gnome.org/local/
+1. click on the "Configure this extension" button.
```
Comment (ba32107) [inline README.md:18]: On Ubuntu 19.10 and up, Chromium is installed as snap by default. As a result, any Chrome extension that use Native Messaging (such as the Gnome extension) does not work. So this settings page is unusable for any user who uses Wintile on Ubuntu (such as myself). Below is a suggestion that includes an alternative approach of configuration:
```suggestion
1. Click on the "Configure this extension" button.
1. Alternatively, open the `Extension` settings in [Gnome Tweaks](https://gitlab.gnome.org/GNOME/gnome-tweaks), locate `Wintile` and click on the cogwheel button to bring up the configuration dialog.
```

**Comment by Fmstrat** [comment]
To merge this, please also add the fact that newer versions of Ubuntu have the "Extensions" application specifically made for configuration. This should probably be the first option, with the others being alternatives.

**Comment by JohnMcLear** [comment]
> To merge this, please also add the fact that newer versions of Ubuntu have the "Extensions" application specifically made for configuration. This should probably be the first option, with the others being alternatives.

I'm on 20.04 and if I search for "Extensions" nothing comes up..  Am I being dumb again?

**Comment by ba32107** [comment]
@JohnMcLear works for me - try it from the command line:
```
$ which gnome-extensions-app 
/usr/bin/gnome-extensions-app
```

**Comment by moemode** [comment]
@JohnMcLear do u have gnome-tweaks installed?

**Comment by JohnMcLear** [comment]
I'm a noob to gnome so probably not.  I'm not at that desktop at current and wont be until mid 2021.  Apologies and thanks for following up <3

**Pull request merged**